database/migrations/2019_07_20_080248_add_locales_to_scheduled_messages.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class AddLocalesToScheduledMessages extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('scheduled_messages', function ($table) {
```

Comment:
One way you can get these migrations working in SQLite is to break them out into separate closures:
```
Schema::table('scheduled_messages', function ($table) {
    $table->text('body_es')->nullable()->after('body');
});

Schema::table('scheduled_messages', function ($table) {
    $table->renameColumn('body', 'body_en');
});
```
but I'm cool will switching to MYSQL too

Reply:
Yeah I think sticking with MySQL might be best, for test/dev/prod parity and all that. Tests are a little slower, and there's a small extra requirement to have a `testing` db, but if using homestead that should be handled now, and CI is working, so I think it's a worthy tradeoff! Good to know on the split closures though, TIL :)